test/test.sh

```diff
@@ -161,11 +161,20 @@ $S3_COMMAND setacl $TEST_BUCKET/aclkey filename=acl
 
 # Test to make sure that it worked
 rm -f acl_new
-echo "$S3_COMMAND getacl $TEST_BUCKET/aclkey filename=acl_new allDetails=1"
-$S3_COMMAND getacl $TEST_BUCKET/aclkey filename=acl_new allDetails=1
+echo "$S3_COMMAND getacl $TEST_BUCKET/aclkey filename=acl_new"
+$S3_COMMAND getacl $TEST_BUCKET/aclkey filename=acl_new
 diff acl acl_new
 rm -f acl acl_new
 
+# Check multipart file upload (>15MB)
```

Comment:
from video chat - multipart stuff here is broken, needs more cleanup.

perhaps drop & push a separate PR for that and then we can get this one in ?